domain/src/test/java/org/mobilitydata/gtfsvalidator/domain/entity/gtfs/transfers/TransferTest.java

```diff
@@ -32,6 +32,8 @@
     private final static String TO_STOP_ID = "stop id 1";
     private final static int VALID_TRANSFER_TYPE_VALUE = 1;
     private final static int VALID_MIN_TRANSFER_TIME_VALUE = 20;
+    private final int UPPER_BOUND_MIN_TRANSFER_TIME = 40;
```

Comment:
Having a hard time
- why there are 4 values total
- why those two are not static like the two others